doc/user/using-graalvm.md

```diff
@@ -16,7 +16,8 @@ information.
 
 By default, GraalVM runs TruffleRuby in the native configuration, which is best
 for short-running processes, but it can also run it on top of the JVM which is
-best for long-running processes. Use `--jvm` to use the JVM configuration.
+best for getting the highest peak performance and for Java interopability. Use
+`--jvm` to use the JVM configuration.
```

Comment:
Maybe it would be good to link to the README here for more details.

Reply:
Fixing.